app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -170,6 +170,18 @@ class NavigationOverlayFragment : Fragment() {
         ).apply {
             onCreateView(view)
         }
+
+        val bannerLayout: View = view.findViewById(R.id.bannerLayout)
+
+        val bannerMoreInfoButton: Button = bannerLayout.findViewById(R.id.bannerMoreInfoButton)
+        bannerMoreInfoButton.setOnClickListener {
+            (activity as MainActivity).onNonTextInputUrlEntered("https://www.mozilla.org")
```

Comment:
This hard coded URL needs to be replaced at a later point.

Reply:
We can land it as is but it'd be ideal to define it as a constant so we don't forget to update it in both places.